.github/workflows/codeql.yml

```
@@ -75,6 +75,21 @@ jobs:
             echo "enabled=true" >> $GITHUB_OUTPUT
           fi
 
+      - name: Setup Rust toolchain
+        if: steps.codeql-check.outputs.enabled == 'true' && matrix.language == 'rust'
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)